### DISCUSSION TITLE
Conditional Breakpoints Not Supported

### DISCUSSION THREAD

**Discussion by JasonZhuAtamation** [forcedotcom/salesforcedx-vscode#5845] [Ideas]
When I use the Apex Replay Debugger to debug Apex code with conditional breakpoints, it does not work.

[a comment by an automated account is not shown]

**Comment by diyer** [comment]
@JasonZhuAtamation We have received similar request from other users. Currently this is not supported and this is an enhancement. We will move this thread under discussions and hopefully add it to our roadmap. Thank you for your feedback!
